accel-pppd/ifcfg.c

```diff
@@ -234,7 +234,11 @@ void __export ap_session_ifdown(struct ap_session *ses)
 			if (!a->installed)
 				continue;
 			if (a->prefix_len > 64)
+#ifdef HAVE_VRF
+				ip6route_del(ses->ifindex, &a->addr, a->prefix_len, NULL, 0, 0, ses->vrf_name);
+#else
 				ip6route_del(ses->ifindex, &a->addr, a->prefix_len, NULL, 0, 0);
+#endif
```

Comment:
there's no need for ifdef all around the code, just use one function with one ifdef inside it